blog/2015/09/instructor-debriefing-2015-09-08-15.html

```diff
@@ -0,0 +1,36 @@
+¬¬¬¬¬ ---
+layout: blog
+root: ../../..
+author: Rayna Harris and Kate Hertweck
+title: "2015 Post-Workshop Instructor Debriefing, Round 17"
+date: 2015-09-08
+time: "08:00:00"
+category: ["Teaching"]
+---
+
+<!-- start excerpt -->
+<p>
+The mentorship committee held their latest round of debriefing for instructors of recent workshops on Tuesday, September 1. Then, on Friday September 4th mentorship committee met for a regular bi-weekly meeting. Some comments from the committee meeting have made their way into this debriefing post.   
+</p>
+<!-- end excerpt -->
+
+<h2> New members on the mentoring committee</h2>
+<p> On August 22, Rainere Silva put out <a href=http://software-carpentry.org/blog/2015/08/join-mentoring-subcommittee.html>a call to join the mentorship committee</a> for members to help with the debriefing session, among other things. The new members include: Rayna Harris, Christina Koch, Sue McClatchy, Mariela Perignon, Phil Rosenfield, Michael Sarahan, and Belinda Weaver. </p>
+
+<h2> Potential changes to the debriefing session format</h2>
+<p> Since March 2015, the debriefing sessions have served to purposes: 1) to debrief instructors of recent workshops, and 2) to give new (or even experienced) instructors an opportunity to hear who the lessons are going and get some tips. The impetus for including new instructors on the call came from <a href=" http://software-carpentry.org/blog/2015/03/instructor-debriefing-2015-03-10.html">a March debriefing session</a> when some novice instructors said they would have benefited from hearing instructor experiences BEFORE doing the instruction themselves. While the dual purpose debriefing session has be working, the mentoring committee is discussing desegregating the two. Stay tuned for more updates regarding specific mentoring opportunities for new instructors.</p>
```

Comment:
"Stay tuned for more updates regarding specific mentoring opportunities for new instructors." -> Could you also invite people to send us suggestions about that?